buildroot-external/rootfs-overlay/usr/lib/systemd/system/raucdb-first-boot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Remove adopted rauc.db on first boot
+After=rauc.service
```

Comment:
Why is this actually `After`? I see that this is the case for `raucdb-update.service` as well, but I am wondering? :thinking: